modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -41,11 +41,11 @@ def submit # rubocop:disable Metrics/MethodLength
           end
           save_auto_claim!(auto_claim)
 
-          if auto_claim.errors.present?
-            raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
-              detail: auto_claim.errors.messages.to_s
-            )
-          end
+          # if auto_claim.errors.present?
+          #   raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
+          #     detail: auto_claim.errors.messages.to_s
+          #   )
+          # end
```

Comment:
is this meant to be commented out?

Reply:
@FonzMP  Good catch I do not think that was meant to stay I will address it now